packages/bundle-size/src/utils/readConfig.test.js

```diff
@@ -0,0 +1,68 @@
+const fs = require('fs').promises;
+const path = require('path');
+const process = require('process');
+const tmp = require('tmp');
+
+const readConfig = require('./readConfig');
+
+/**
+ * @param {string} configContent
+ * @param {number} pwdNesting How many folders to nest the pwd
+ * @return {Promise<string>} Returns a relative path to a config file
+ */
+async function setup(configContent, pwdNesting = 0) {
+  let packageDir = tmp.dirSync({ prefix: 'test-package', unsafeCleanup: true });
+  const config = tmp.fileSync({ dir: packageDir.name, name: 'bundle-size.config.js' });
+
+  for (let i = 0; i < pwdNesting; i++) {
+    packageDir = tmp.dirSync({ dir: packageDir.name, prefix: 'nested' });
```

Comment:
```suggestion
    packageDir = tmp.dirSync({ dir: packageDir.name, prefix: 'nested', unsafeCleanup: true });
```

@ecraig12345 mentioned in one of previous PRs that `unsafeCleanup: true` is required for each call of `.dirSync()`, https://github.com/microsoft/fluentui/pull/18322#discussion_r656421425